src/lib/bootloader/grub2.rb

```diff
@@ -121,6 +151,14 @@ def packages
 
   private
 
+    def replace_with(other)
+      stage1.clear_devices
```

Comment:
well, I am not sure. Originally I plan to have it in CFA, but then I found that there are more strategies for merge. So I move it to given bootloader itself to drive its merge strategy.